06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.html

```diff
@@ -0,0 +1,164 @@
+                  <span ng-if="!indexDisplay">Search result for "{{searchQuery}}" in "{{details.searchedFile}}"</span>
+                </h2>
+                <table class="table table-striped" id='myTable'>
+                  <thead>
+                    <th>Tokens</th>
+                    <th style="text-align:center" ng-repeat="i in details.documents">Document {{i+1}}</th>
+                  </thead>
+                  <tbody>
+                    <tr ng-repeat="(key, value) in details.indexes">
+                      <td>{{key}}</td>
+                      <td ng-repeat="docnum in details.documents" style="text-align:center; font-weight:bold">
+                        <span ng-if="value.includes(docnum)">X</span>
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+            </div>
+            <br/>
+        </div>
+        <div class="col-sm-4">
+            <blockquote>
+              <h1>
+                About DPlex
+              </h1>
+                <p>
+                  DPlex in an inverted index application. DPlex allows you to
+                  upload a book in JSON format. You can index the document and
+                  search for occurences of word(s) in a document or entirely
+                  uploaded documents.
+                </p>
+
+                <p>
+                  <h3>How to use</h3>
+                  <ol>
+                    <li>To Create indexes of a file
+                        <ul>
+                          <li>Select a file in JSON format and the structure
+                            below</li>
+                          <li>Select the document to index from the list of
+                            files and click Create Index</li>
+                        </ul>
+                    </li>
+                    <li>Search Files
+                        <ul>
+                          <li>Enter the search terms which could either be a single word or combination of words.</li>
+                          <li>Select which of the indexed file you want to search. You can also search files.</li>
+                        </ul>
+                    </li>
+                  </ol>
+                </p>
+            </blockquote>
+            <blockquote>
+              Sample JSON Format to be uploaded can be found <a href='https://gist.github.com/q-ode/72019451b98f079a8d737eb7a412bf14' target='_blank'>here</a>
+            </blockquote>
+
+        </div>
+    </div>
+</body>
+  <script type="text/javascript" src="src/module.js"></script>
+  <script src="https://unpkg.com/angular-toastr/dist/angular-toastr.tpls.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script type="text/javascript" src="src/invertedindex.js"></script>
+  <script type="text/javascript" src="src/app.js"></script>
+</html>
```

Comment:
Depending on the number of documents in the uploaded file, will the table of indices created be responsive? Will the table overflow out of its width? Check that also. If you are not sure what this means please see your peers to get a better understanding.